EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -48,47 +45,16 @@ private void DoFalseDiscoveryRateAnalysis(FdrAnalysisResults myAnalysisResults)
             {
                 var psms = proteasePsms.ToList();
 
-                // generate the null distribution for e-value calculations
-                double globalMeanScore = 0;
-                int globalMeanCount = 0;
-
-                if (CalculateEValue && psms.Any())
-                {
-                    List<double> combinedScores = new List<double>();
-
-                    foreach (PeptideSpectralMatch psm in psms)
-                    {
-                        psm.AllScores.Sort();
-                        combinedScores.AddRange(psm.AllScores);
-
-                        //remove top scoring peptide
-                        if (combinedScores.Any())
-                        {
-                            combinedScores.RemoveAt(combinedScores.Count - 1);
-                        }
-                    }
-
-                    if (combinedScores.Any())
-                    {
-                        globalMeanScore = combinedScores.Average();
-                        globalMeanCount = (int)((double)combinedScores.Count / psms.Count);
-                    }
-                    else
-                    {
-                        // should be a very rare case... if there are PSMs but each PSM only has one hit
-                        globalMeanScore = 0;
-                        globalMeanCount = 0;
-                    }
-                }
-
-                //Calculate delta scores for the psms (regardless of if we are using them)
-                foreach (PeptideSpectralMatch psm in psms)
-                {
-                    if (psm != null)
-                    {
-                        psm.CalculateDeltaScore(ScoreCutoff);
-                    }
-                }
+                ////Calculate delta scores for the psms (regardless of if we are using them)
```

Comment:
@zrolfs need to talk about this code, we changed some delta score stuff

Reply:
requested zach to look at it.